src/fromager/overrides.py

```diff
@@ -69,8 +70,9 @@ def extra_environ_for_pkg(
     Extra environment variables are stored in per-package .env files in the
     envs package, with a key=value per line.
 
+    By default all packages have access to the environment variables in the current process
     """
-    extra_environ = {}
+    extra_environ = dict(os.environ)
```

Comment:
This looks like a reasonable approach, but it makes the name of this function seem wrong. The return value is no longer "extra" values, it's all of the values.

A similar change at https://github.com/python-wheel-build/fromager/blob/main/src/fromager/wheels.py#L114 would keep the intent of this function intact. The trade-off of that is that any plugin overriding the `build_wheel` function would have to do this step, too. Maybe that's reasonable, because perhaps one reason to override is to _not_ share some variable?

What do you think?

Reply:
That makes sense